cmd/ddev/cmd/composer-create.go

```diff
@@ -119,22 +117,26 @@ project root will be deleted when creating a project.`,
 			composerCmd = append(composerCmd, "--no-interaction")
 		}
 
+		if preferDistArg {
+			composerCmd = append(composerCmd, "--prefer-dist")
+		}
+
 		composerCmdString := strings.TrimSpace(strings.Join(composerCmd, " "))
 		output.UserOut.Printf("Executing composer command: %s\n", composerCmdString)
 		stdout, _, err := app.Exec(&ddevapp.ExecOpts{
 			Service: "web",
 			Cmd:     composerCmd,
 		})
 		if err != nil {
-			util.Failed("Failed to execute create-project command")
+			util.Failed("Failed to create project")
```

Comment:
Let's at least add the error code. If the actual output is being reported elsewhere then that's ok.

Reply:
util.Failed though will mean that the existing directories don't get cleaned up. So we should probably defer the directory cleanup instead of doing it inline?